### PR TITLE
add historical updates page (corrigendum)

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This page adds all of the corrigendum items. Made the items #### headers so each item is not too big but also easy to link to.

Created button on home page to link to this page as it won't be shown in the left-side navigation menu: https://github.com/PureStake/moonbeam-mkdocs/pull/97

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

N/A

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]
Looks good. I would organize the Bugs and Migrations by Date and add the block number of each RT upgrade for reference.

Thanks for putting this together!

**Comment by eshaben** [comment]
@albertov19 I know you already approved, but with the large change I just want to make sure it looks good to you before merging

**Review by albertov19 — APPROVED** [review]
Looks great! I know this entailed an humongous effort, thank you!

File: builders/build/historical-updates.md [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+
+### XCM-Related Pallets {: #xcm-related-pallets }
+
+#### Update Transact Info Storage Item {: #update-transaction-info }
+
+There was a migration applied to the `TransactInfo` storage item of the XCM-transactor pallet that changed the following items:
+
+- `max_weight` is added to prevent transactors from stalling the queue in the destination chain
+- Removes `fee_per_byte`, `metadata_size`, and `base_weight` as these items are not necessary for XCM transactions
+- `fee_per_second` replaces `fee_per_weight` to better reflect cases (like Kusama) in which the `fee_per_eight` unit is lower than one
```
Comment (wuxinxin2000) [inline builders/build/historical-updates.md:453]: `fee_per_eight` should be `fee_per_weight`?

Comment (eshaben) [reply builders/build/historical-updates.md:453]: Fixed in https://github.com/PureStake/moonbeam-docs/pull/519

**Pull request merged**